packages/lifecycle/README.md

```diff
@@ -0,0 +1,125 @@
+<p>
+  <img width="100%" src="https://assets.solidjs.com/banner?type=Primitives&background=tiles&project=lifecycle" alt="Solid Primitives lifecycle">
+</p>
+
+# @solid-primitives/lifecycle
+
+[![turborepo](https://img.shields.io/badge/built%20with-turborepo-cc00ff.svg?style=for-the-badge&logo=turborepo)](https://turborepo.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/lifecycle?style=for-the-badge&label=size)](https://bundlephobia.com/package/@solid-primitives/lifecycle)
+[![version](https://img.shields.io/npm/v/@solid-primitives/lifecycle?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/lifecycle)
+[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsolidjs-community%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/solidjs-community/solid-primitives#contribution-process)
+
+Package providing extra layer of lifecycle primitives for Solid.
+
+- [`createIsMounted`](#createIsMounted) - Returns a boolean signal indicating whether the component is mounted or not.
+- [`isHydrated`](#isHydrated) - A signal with the same behavior as [`isHydrating`](#isHydrating) but this one focused only on client-side updates.
+- [`onConnect`](#onConnect) - Calls the given callback when the target element is connected to the DOM.
```

Comment:
This name might lead to some confusion with the connectivity package. `onReady` maybe?

Reply:
I'm not sure
connect is related to the element being connected to the DOM, if you know about the `el.isConnected` field, you'll understand this one.
if they were in the same package - sure, but they are separated for a reason - component/reactivity lifecycle has nothing to do with internet connection

Reply:
so it's probably the connectivity package that is wrongly named. for "connect" to have meaning it needs to have context

Reply:
I guess I could name it `onElementConnect` to give that missing context here 🤔

Reply:
Maybe `onConnectNode`?